examples/meta/src/preprocessor/logplusone.sg

```diff
@@ -0,0 +1,30 @@
+File data_train = csv_file("../../data/fm_train_real.dat")
```

Comment:
rename to "log_plus_one" ... you get the point of how we want the filenames to look like :)